controllers/cloudinit/controlplane_init.go

```diff
@@ -149,12 +153,28 @@ func NewInitControlPlane(input *ControlPlaneInput) ([]byte, error) {
 	input.WriteFiles = append(input.WriteFiles, input.AdditionalFiles...)
 	input.SentinelFileCommand = sentinelFileCommand
 	input.ControlPlaneEndpointType = "DNS"
+	major, minor, err := extractVersionParts(input.Version)
+	if err != nil {
+		return nil, err
+	}
+	input.Version = generateSnapChannelArgument(major, minor)
```

Comment:
why not simply allowing the user to work with the snap channel?
This way we can test `latest/edge` and not just `1.25`-like channels

Reply:
This is a good point. I am going to create a task-item for us to add the channel as an option a user can set.